R/src_dbi.R

```diff
@@ -90,9 +90,13 @@
 #'   summarise(n = n()) %>%
 #'   show_query()
 #' }
-src_dbi <- function(con, auto_disconnect = FALSE) {
+src_dbi <- function(con, auto_disconnect = TRUE) {
```

Comment:
This can't be `TRUE` because we create a bunch of transient `src_dbi`s when working directly with DBI connections

Reply:
Or at least we need to change to this in dplyr:

```R
#' @export
tbl.DBIConnection <- function(src, from, ...) {
  check_dbplyr()
  tbl(dbplyr::src_dbi(src), from = from, ..., auto_disconnect = FALSE)
}
```

Reply:
Sent PR to dplyr. A coarse CRAN search suggests that the default for `src_dbi()` should be `auto_disconnect = FALSE`, but only from the next dplyr release on.

Reply:
I rolled it back in dbplyr, I'm pretty sure defaulting to `FALSE` is correct - you should set to `TRUE` only when creating the connection in the `src_dbi()` call, or when you know the scope of the source and the connection are identical.